### PR TITLE
Prepare for 2.0.0-preview.3

### PR DESCRIPTION
**Pull request by rjmholt**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+#### [vscode-powershell](https://github.com/powershell/vscode-powershell)
+
+- [vscode-PowerShell #1852](https://github.com/PowerShell/vscode-PowerShell/pull/1852) -
+  Turn `powershell.codeformatting.useCorrectCasing` setting off by default until PSSA issues are fixed (Thanks @bergmeister!)
+- [vscode-PowerShell #1838](https://github.com/PowerShell/vscode-PowerShell/pull/1838) -
+  Set PSReadLine featureFlag default to null so that it can be resolved by settings
+- [vscode-PowerShell #1837](https://github.com/PowerShell/vscode-PowerShell/pull/1837) -
+  Do not use -EncodedCommand on Windows
+- [vscode-PowerShell #1825](https://github.com/PowerShell/vscode-PowerShell/pull/1825) -
+  Switch to current lowercase names for powershell and mdlint recommended extensions (Thanks @rkeithhill!)
```
Comment (bergmeister) [inline CHANGELOG.md]: What about the WhitespaceAroundPipe and WhitespaceBeforeBrace settings? I feel the settings are so new we could risk the breaking change of having them lower-cased

Comment (rjmholt) [reply CHANGELOG.md]: I think we should do that too

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "powershell",
   "displayName": "PowerShell",
-  "version": "2.0.1",
+  "version": "2.0.3",
```
Comment (TylerLeonhardt) [inline package.json]: change to 2.0.2

File: tools/releaseBuild/Image/DockerFile [hunk 3]
```diff
@@ -23,7 +23,7 @@ COPY build.ps1 containerFiles/build.ps1
 
 # Add an environment variable for build versioning
 ENV VSTS_BUILD=1
-ENV VSTS_BUILD_VERSION=2.0.1
+ENV VSTS_BUILD_VERSION=2.0.3
```
Comment (TylerLeonhardt) [inline tools/releaseBuild/Image/DockerFile]: 2.0.2

File: package.json (hunk 2 above)
Comment (TylerLeonhardt) [reply package.json]:
```suggestion
  "version": "2.0.2",
```

File: tools/releaseBuild/Image/DockerFile (hunk 3 above)
Comment (TylerLeonhardt) [reply tools/releaseBuild/Image/DockerFile]:
```suggestion
ENV VSTS_BUILD_VERSION=2.0.2
```

**Pull request merged**